#ifndef _3RD_HEADER_
#define _3RD_HEADER_

#pragma GCC system_header
#include <mavros_msgs/State.h>
#include <std_msgs/Float64.h>
#include <std_msgs/String.h>
#include <cstring>
#include <geometry_msgs/PoseStamped.h>
#include <mavros_msgs/ParamGet.h>
#include <mavros_msgs/SetMode.h>
#include <mavros_msgs/CommandBool.h>
#include <string>
#include <iostream>
#include <fstream>
#include <sstream>
#include <mavros_msgs/CommandTOL.h>
#include <keyboard/Key.h>
#include <tf/tf.h>
#include <tf/transform_datatypes.h>
#include <nav_msgs/Odometry.h>
#include "Eigen/Eigen"
#include "bearing_common/AllPosition.h"
#include "geometry_msgs/TwistStamped.h"
#include "bearing_common/HParamSetSrv.h"
#include  "unsupported/Eigen/KroneckerProduct"
#include <mavros_msgs/CommandBool.h>
#include <sstream>
#include <geometry_msgs/Vector3.h>
#include <gazebo_msgs/ModelStates.h>
#include <geometry_msgs/Pose.h>
#include <math.h>

#include "bearing_common/Bearing2D.h"
#include "bearing_common/GroupBearing.h"
#include "bearing_common/DataRecord.h"
#include "bearing_common/TriggerRec.h"
#include "geometry_msgs/TransformStamped.h"

#endif
